### ISSUE TITLE
No Activation Email

### ISSUE THREAD

**Issue by Xmaster6726** [nathan815/TheSwanStation-Issues#73]
I've tried to make multiple accounts with different emails now, and none of them send an email to me.

**Comment by nathan815** [comment]
Hi @Xmaster6726. Sorry for my super late response - just now saw this. Please let me know your username and I can active it manually.

**Comment by Maldevar** [comment]
Can confirm also if you are gonna active accounts manually my username is "Deppresso drip"

**Comment by nathan815** [comment]
@Deppresso-Exppresso  Done :)

**Comment by Maldevar** [comment]
Also generally all things involving email doesn't seem to work

**Comment by nathan815** [comment]
Yeah, it's not super reliable - it works for some people and not others unfortunately. I am going to see if I can get it fixed, but am currently rebuilding from scratch and plan to release it in February. Will be using SendGrid or some other proper service for emails 😆

**Comment by Al-AJ52** [comment]
@nathan815 Same situation as well.
I've made with 2 accounts with different emails, thought it was the other email problem whatnot.
So these 2 usernames are mine; **AJ#2342** (_Gmail_) & **WandererAJ** (_Yahoo_)
Much appreciated if the situation is resolved.
Thanks.

**Comment by nathan815** [comment]
@Al-AJ52  Sorry about that, I've just activated both of those usernames!

I am disabling the email system for now.

**Comment by nathan815** [comment]
@Al-AJ52  Sorry about that, I've just activated both of those usernames!

**Comment by covertcacoethes** [comment]
@nathan815 Currently suffering the same fate- no email authentication received for account activation; registered username is homonymous: _covertcacoethes_

**Comment by nathan815** [comment]
@covertcacoethes Please try logging in, every account is auto-activated currently due to the email issue.
